mkdocs_blogging_plugin/plugin.py

```diff
@@ -337,9 +337,9 @@ def with_timestamp(self, page, by_revision):
         timestamp = None
         if "time" in page.meta:
             timestamp = self._parse_time(page.meta["time"])
-        if "date" in page.meta and timestamp is None:
+        elif "date" in page.meta:
             timestamp = self._parse_time(page.meta["date"])
-        if not timestamp:
+        else:
```

Comment:
If `time` is included in the page meta but is _not_ a date time, and `meta_time_format` is not set, this will end up with `timestamp = None`.

For example:

```
---
time: 5 minutes
date: 2024-04-12
---
```

The `time` meta could be meant for another plugin.

Using `is None` would try with `"date"` if `"time"` is present but cannot be parsed.

Reply:
Okay, I understand. Thank you for the explanation.